widgets/VersionList.php

```diff
@@ -60,6 +63,15 @@ public function onSearch()
         return $this->updateList();
     }
 
+    public function onSort()
+    {
+        $this->config->sort = request()->sort;
```

Comment:
@filipac We prefer using the facade for reading data.
```suggestion
        $this->config->sort = Request::input('sort');
```